tools/mo/openvino/tools/mo/utils/cli_parser.py

```diff
@@ -1072,7 +1072,9 @@ def get_freeze_placeholder_values(argv_input: str, argv_freeze_placeholder_with_
             plh_with_value = plh_with_value.split('->')
             if len(plh_with_value) != 2:
                 raise Error("Wrong replacement syntax. Use --freeze_placeholder_with_value "
-                            "\"node1_name->value1,node2_name->value2\"")
+                            "\"node1_name->value1,node2_name->value2\""
+                            " or for multiple values --freeze_placeholder_with_value " 
+                            "\"node1_name->[value1 value2],node2_name->[value3 value4]\"")
```

Comment:
I would recommend do not touch `--freeze_placeholder_with_value` because it is deprecated for some time. And now we use `--input` option for freezing instead

Reply:
This is deprecated option. Please do not update it.